HVZ.Persistence.MongoDB/Repos/GameRepo.cs

```diff
@@ -71,7 +77,7 @@ public void InitIndexes()
         });
     }
 
-    public async Task<Game> CreateGame(string name, string creatorid, string orgid)
+    public async Task<Game> CreateGame(string name, string creatorid, string orgid, int maxOzTags = 3)
```

Comment:
is this the right place to define the default number of `maxOzTags`? shouldn't this usually come from an org setting or from some setting field at the time of game creation?